projects/plugins/jetpack/extensions/blocks/recipe/recipe-print.js

```diff
@@ -0,0 +1,87 @@
+const printBtns = document.querySelectorAll(
+	'.wp-block-jetpack-recipe-details__detail .wp-block-button__link'
+);
+const style = document.createElement( 'style' );
+
+printBtns.forEach( btn => {
+	btn.addEventListener( 'click', () => {
+		style.id = 'jetpack-recipe-block-print-style';
+
+		style.innerHTML = `
+      @media print {
```

Comment:
Instead of including a js file, could we add that CSS to the existing stylesheet that gets enqueued with the block?

Reply:
@jeherve - The only reason I chose to do JS here was due to how aggressive the print stylesheet needed to be on pages that include the Recipe block. I'm only applying these styles on click of the Print button so as to not totally take over the print styles of the entire page that a Recipe resides on.

This was an attempt for the Recipe block to play nicely within a site that may also require full pages to be printed out.

If you think that the recipe block warrants overtaking other print styles on the page entirely then I can certainly rip the JS out in favor of a CSS enqueue!

Reply:
That's a good point. I previously assumed that we'd always want the recipe to be the only thing printed whenever there is a recipe on the page, but that may be a wrong assumption, you're right. Let's keep adding the styles dynamically, that's a good idea.

That said, I wonder if instead of loading a separate file, we could move all this to the existing `view.js` file, to keep everything in one place and not have to enqueue another file.